nautobot/extras/templates/extras/jobresult.html

```diff
@@ -100,6 +100,14 @@
                         {% include 'extras/inc/json_data.html' with data=result.data format="json" %}
                     </div>
                 </div>
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <strong>Traceback</strong>
+                    </div>
+                    <div class="panel-body">
+                        {% include 'extras/inc/json_data.html' with data=result.traceback format="python" %}
```

Comment:
Wondering if we should do a check for settings.DEBUG here - similar to how we only present full tracebacks for other exceptions (500 errors) if it's True?

Reply:
IMO if there is a traceback we should show it. I was thinking that since it's stored on the object, presenting it if it exists is easy. Is there a concern about data exposure or something tangible?

Reply:
Yeah, that was my thinking.

Reply:
That's valid. IMO let's leave that as a v2 UI thing. I've tracked it to add to the backlog.